schemas/appeal-s78.schema.json

```diff
@@ -0,0 +1,1078 @@
+    },
+    "affectsScheduledMonument": {
+      "type": ["boolean", "null"],
+      "description": "Indicates that a scheduled monument is affected",
+      "$comment": "Only available in S78 cases"
+    },
+    "hasProtectedSpecies": {
+      "type": ["boolean", "null"],
+      "description": "Indicates the existence of protected species",
+      "$comment": "Only available in S78 cases"
+    },
+    "isAonbNationalLandscape": {
+      "type": ["boolean", "null"],
+      "description": "Indicates an area of outstanding beauty (National Landscape)",
+      "$comment": "Only available in S78 cases"
+    },
+    "designatedSitesNames": {
+      "type": ["string", "null"],
+      "description": "The designated site names"
+    },
+    "hasTreePreservationOrder": {
+      "type": ["boolean", "null"],
+      "description": "Indicates a TPO (tree preservation order)",
+      "$comment": "Only available in S78 cases"
+    },
+    "isGypsyOrTravellerSite": {
+      "type": ["boolean", "null"],
+      "description": "Indicates a traveller site",
+      "$comment": "Only available in S78 cases"
+    },
+    "isPublicRightOfWay": {
+      "type": ["boolean", "null"],
+      "description": "Indicates a PROW (public right of way)",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaEnvironmentalImpactSchedule": {
+      "type": ["string", "null"],
+      "description": "The impact schedule from EIA",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaDevelopmentDescription": {
+      "type": ["string", "null"],
+      "description": "The development description from EIA",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaSensitiveAreaDetails": {
+      "type": ["string", "null"],
+      "description": "The sensitive area details from EIA",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaColumnTwoThreshold": {
+      "type": ["boolean", "null"],
+      "description": "EIA specific threshold",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaScreeningOpinion": {
+      "type": ["boolean", "null"],
+      "description": "EIA screening opinion",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaRequiresEnvironmentalStatement": {
+      "type": ["boolean", "null"],
+      "description": "EIA environmental statement required",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaCompletedEnvironmentalStatement": {
+      "type": ["boolean", "null"],
+      "description": "EIA completed environmental statement required",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaHasStatutoryConsultees": {
+      "type": ["boolean", "null"],
+      "description": "EIA statutory consultees",
+      "$comment": "Only available in S78 cases"
+    },
+    "eiaConsultedBodiesDetails": {
+      "type": ["string", "null"],
+      "description": "The details of the bodies consulted by EIA"
+    },
+    "eiaHasConsultationResponses": {
```

Comment:
is this eia?

Reply:
I forgot to ask Gareth tbh... could you ping him?

Reply:
hasConsultationResponses?